java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -595,10 +595,10 @@ void testReadORCFull() {
   @Test
   void testReadORCNumPyTypes() {
     assumeTrue(Cuda.isEnvCompatibleForTesting());
-    // by default ORC will promote date and timestamp columns to DATE64
+    // by default ORC will promote DATE32 to DATE64
```

Comment:
nit, I feel this comment could say something about `TIMESTAMP`. Is it that now `DATE32` is promoted to `TIMESTAMP`?